app/views/styleguide/pages/locator_invalid.erb

```diff
@@ -3,21 +3,23 @@
 <div class="l-grid-row">
   <div class="l-column-two-thirds">
     <main role="main" id="content" tabindex="-1">
-      <h1>Locations near you</h1>
-      <p><strong>SWXJ</strong> is not a valid post code. Please try again.</p>
+      <h1>Face-to-face appointments near sw</h1>
```

Comment:
Is it advisable to include user input when we know its not a valid postcode?

It opens the page up to abuse.